src/chapters/angular-lsn1/studio.rst

```diff
@@ -1,62 +1,197 @@
+   #. When prompted about using routing, enter "N" for No.
+   #. When prompted to select the stylesheet format, select CSS.
 
-We will create a Space Mission Planning Dashboard.
+::
 
-This Angular project needs to include five sections:
-    #. A header that displays the mission and rocket name,
-    #. A list that displays the names of the crew,
-    #. A list that displays crew member roles,
-    #. A list that displays required equipment,
-    #. A list that displays the countries represented by the crew.
+   your-computer:angular-lc101-mission-planner $ ng new mission-planning-dashboard
+   ? Would you like to add Angular routing? No
+   ? Which stylesheet format would you like to use? CSS
 
-Each section needs to be it's own component, and they all need to be placed
-within the ``app`` folder.
+6. Navigate into the new folder by running ``cd mission-planning-dashboard``
+7. Install dependencies by running ``npm install``
+8. Verify that the application will run by running ``ng serve``
+9. View the site in your browser at http://localhost:4200
 
-In the next two studios we will learn how to expand this project by using more
-Angular tools that will make the application more interactive. For the purposes
-of today's studio our web application will not be interactive, and will simply
-display static information.
+   * You should see a header that says "Welcome to mission-planning-dashboard!"
 
-Project Data
+10. Stage and commit the files before starting on the features.
+
+Requirements
 ------------
+The mission dashboard you are creating will eventually look like this.
 
-Data you will need for this project:
-   #. Mission name: Apollo 11
-   #. Rocket name: Saturn V
-   #. Crew member names & roles: Commander Neil Armstrong, Command Module Pilot
-      Michael Collins, Lunar Module Pilot Buzz Aldrin
-   #. Equipment: Command Module, Lunar Module, 3 flight suits, 2 moon suits, 30
-      meals, 100 litres of water
-   #. Countries: United States
+.. figure:: ./figures/example-mission-dashboard.png
+   :alt: Screen shot showing the mission dashboard with mission name, rocket name, crew members, equipment, and experiments.
 
-If you need a reminder on how to create components and organize them using the
-app component work through the :ref: `` again.
+Update Starter Page Content
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The default stater page created by Angular contains default text, images, and links. Your job is to remove the default content.
 
-Bonus Mission
--------------
+1. Clear out the contents of file ``mission-planning-dashboard/src/app/app.component.html``.
+2. Type in the text ``Add components here...`` into ``app.component.html``.
+3. View the app in your browser to verify the words "Add components here..." is the only thing that appears on the page.
+
+Header Component
+^^^^^^^^^^^^^^^^
+You need to create a component that shows the title, mission name, and carrier rocket.
+
+1. Create a header component using ``ng g component header``.
```

Comment:
We should remind the students to use the terminal to navigate into the ``app`` folder before generating components.